packages/manager/modules/anthos/src/dashboard/components/order-public-ip/constants.js

```diff
@@ -0,0 +1,15 @@
+export const ADD_ON_PLAN_CODE = 'option-public-ips-64';
```

Comment:
I think could be write in one word `ADDON_PLAN_CODE `

Reply:
:+1: